src/ansys/motorcad/core/geometry.py

```diff
@@ -552,7 +552,7 @@ def rotate(self, centre_point, angle):
         Parameters
         ----------
         centre_point : Coordinate
-            point to rotate Coordinate around.
+            Point to rotate Coordinate around.
```

Comment:
```suggestion
            Point to rotate coordinate around.
```
Only capitalize proper nouns or the first letter of the initial word (as you did).